lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -148,6 +149,15 @@ def symlink_plugin_paths(source_dir, target_path)
           end
         end
       end
+
+      def compile_locale_files
+        Dir[miq_dir.join('locale/*/*.po')].each do |po|
+          mo = "#{File.dirname(po)}/LC_MESSAGES/#{File.basename(po, '.po')}.mo"
+          shell_cmd("mkdir #{File.dirname(mo)} ; msgfmt #{po} -o #{mo}")
```

Comment:
I guess `msgfmt` comes from `gettext`.  Do we need to add that to the list of packages in the Dockerfile?

https://github.com/ManageIQ/manageiq-rpm_build/blob/10200c9129542cf0585da2b210cc189bcdb2601b/Dockerfile#L14-L46

Reply:
Oh yeah good catch

Reply:
Huh. wonder why it is working though.

Reply:
I don't know if something else is enabling `msgfmt`.  Maybe try to see if you can get it to fail and complain about a missing `msgfmt` first, then we can try to enable whatever module that provides it.  It should fail right now unless something else is making it available as a direct or indirect dependency

Reply:
yea, I had already tested this a number of times (building the rpms / upgrading the appliance. not a brand new appliance)

```
[root@localhost ~]# rpm -qf /usr/bin/msgfmt
gettext-0.19.8.1-17.el8.x86_64
[root@localhost ~]# rpm -q --whatrequires gettext
grub2-tools-minimal-2.02-99.el8.x86_64
grub2-tools-2.02-99.el8.x86_64
grub2-tools-extra-2.02-99.el8.x86_64
grub2-pc-2.02-99.el8.x86_64
```

Looks like `grub2` sepends upon `gettext` -- so this is not going anywhere.

Reply:
So, to answer my quesion:

> I guess `msgfmt` comes from `gettext`. Do we need to add that to the list of packages in the Dockerfile?

No, w can't get very far without grub2 and grub2 depends on gettext which provides the command we're trying to run so we don't have to explicitly require the gettext rpm.